pkg/domain/infra/abi/system.go

```
@@ -361,6 +361,12 @@ func (ic *ContainerEngine) SystemDf(ctx context.Context, options entities.System
 	}, nil
 }
 
+func (ic *ContainerEngine) SystemGc(ctx context.Context) (*entities.SystemGcReport, error) {
+	fmt.Printf("Running SystemGc\n")
+	_ = ic.Libpod.GarbageCollect()
```

error is ignored here